tests/plugins/test_add_filesystem.py

```diff
@@ -407,12 +445,11 @@ def test_image_build_overwrites(tmpdir):
                     [fedora-os]
                     baseurl = http://default-repo.com/fedora/$basearch/os.repo
                     """))
-    plugin = create_plugin_instance(tmpdir, {'repos': repos})
+    plugin = create_plugin_instance(tmpdir, {'repos': repos, 'architectures': architectures})
     image_build_conf = dedent("""\
         [image-build]
         name = my-name
         version = 1.0
-        arches = i386,i486
```

Comment:
You could enhance this test to:

- Ensure arches when defined here are overwritten by `architectures` plugin parameter
- Ensure arches when defined here are overwritten by `architecture` plugin parameter
- Ensure that if neither `architecture` nor `architectures` is provided, this arches value is used